tests/sources/fixtures/onedrive/fixture.py

```diff
@@ -0,0 +1,270 @@
+#
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License 2.0;
+# you may not use this file except in compliance with the Elastic License 2.0.
+#
+"""Module to handle api calls received from connector."""
+
+import io
+import os
+import random
+import string
+import time
+
+from faker import Faker
+from flask import Flask, make_response, request
+from flask_limiter import HEADERS, Limiter
+from flask_limiter.util import get_remote_address
+
+app = Flask(__name__)
+
+
+DATA_SIZE = os.environ.get("DATA_SIZE", "small").lower()
+
+match DATA_SIZE:
+    case "extra_small":
+        TOTAL_USERS = 50
+        FILE_COUNT_PER_USER = 100
```

Comment:
Let's get rid of this

Reply:
(for context, I'm working on improving ftests and never had a need for extra_small)